dynamic_search.py

```
@@ -0,0 +1,163 @@
+    def get_publisher(self, comic_id: str) -> str:
+        self.cursor.execute(
+            """
+            SELECT p.name
+            FROM comics c
+            LEFT JOIN publishers p ON c.publisher_id = p.id
+            WHERE c.id = ?
+            """, (comic_id,)
+        )
+        return self.cursor.fetchone()[0]
+
+    def rebuild_fts5(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `rebuild_fts5`

Add return type annotation: `None`

(ANN201)